Automattic-Tracks-iOS/Crash Logging/SentryTestError.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+public struct SentryTestError: LocalizedError {
+    public let title: String
+    public let code: Int
+
+    public init(title: String, code: Int = -1) {
+        self.title = title
+        self.code = code
+    }
+}
```

Comment:
After some investigation and playing with the code, the String `The operation couldn't be completed.` comes from when an `error.localizedDescription` of any `Swift.Error` doesn't provide a custom implementation.
Which is the case here, and explains why we don't see the details of the error title nor code in the Sentry event.

I've modified the code locally to implement one, and it indeed solved it and finally made the `title` and `code` of the error finally surface and be visible in Sentry.

```suggestion
    public var errorDescription: String? {
        "\(title) (code \(code))"
    }
}
```

Reply:
Ah sorry, yeah as you now know we were hitting that path with every test error 😅

I just happened to know what to look for in Sentry.

I like this for its obviousness in these situations.

Addressed in 9b9c107b2cacca1cd986d31619366a36a341a414 – I just added a bit of a description to the `errorDescription` field